### PR TITLE
Added webpack config (to get the UI to work in stage)

### PR DESCRIPTION
**Pull request by justinorringer**
## What?
- Following payload-trackers lead, the app will use webpack to load all assets.

So here I added the necessary libraries and fixed compilation errors -- the thing didn't like the pathing :(.

## Why?
The UI showing up in stage would be great.

## How?
A bunch of new loaders and webpack packages.

## Testing
I ran it with the webpack locally, and it worked!

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] General Coding Practices
- [ ] File Management
- [ ] Memory Management

### CODE REVIEW

File: src/components/TimelineCard.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ export const TimelineCardWrapper = React.forwardRef((props, ref) => {
...
-    includeRepo: boolean;
-    ghURL: string;
-    glURL: string;
-}
+// type TimelineCardProps = {
+//     timeline: any;
+//     includeRepo: boolean;
+//     ghURL: string;
+//     glURL: string;
+// }
```
Comment (CodyWMitchell) [inline src/components/TimelineCard.js:19]: Do we no longer need the type checking for includeRepo, ghURL, and glURL here?

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -18,16 +18,3 @@ jobs:
       steps:
         - name: Checkout Code
           uses: actions/checkout@v2
-
-        # Initializes the CodeQL tools for scanning.
```
Comment (CodyWMitchell) [inline .github/workflows/pr.yml:22]: Was CodeQL not being used for anything? Wanted to check before we remove it.

Comment (justinorringer) [reply .github/workflows/pr.yml:22]: Yup! It has never done anything with the analysis, and I've been meaning to remove it. Didn't remember removing it honestly haha.

File: src/components/TimelineCard.js (hunk 1 above)
Comment (justinorringer) [reply src/components/TimelineCard.js:19]: They never _needed_ to be checked, but I liked how explicit it was. The data comes from the api and then is mapped to each card, so I think its pretty safe.

The babel-loader though does not recognize type definitions, so I removed it and 'cast' it on line 22. I wish there was a way to make it less picky.

File: src/AppConstants.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+export const API_URL = process.env.ENV === 'development' ? 'http://localhost:8000' : '/app/payload-tracker';
```
Comment (dehort) [inline src/AppConstants.js:1]: Does this need to be payload-tracker or changelog?

**Review by dehort — APPROVED** [review]
LGTM!  (I don't really know what i'm looking at!!  ;)  )

File: src/AppConstants.js (hunk 3 above)
Comment (justinorringer) [reply src/AppConstants.js:1]: oh gosh thanks!!!

File: src/components/TimelineCard.js (hunk 1 above)
Comment (CodyWMitchell) [reply src/components/TimelineCard.js:19]: Sounds good!

**Review by CodyWMitchell — APPROVED** [review]
LGTM!

**Pull request merged**